knowledge/laws/traffic_laws_us/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: traffic_laws
+seed_examples:
+- answer: | Pedestrian crossings are strips of the road where pedestrians are expected to cross
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!